cleo_sdk/CLEO_Utils.h

```diff
@@ -117,6 +117,38 @@ namespace CLEO
         return std::move(info);
     }
 
+    // does file path points inside game directories? (game root or user files)
+    static bool IsFilepathSafe(CLEO::CRunningScript* thread, const char* path)
+    {
+        // TODO: path handling should use more advanced utils from std::filesystem
```

Comment:
maybe call [canonical](https://en.cppreference.com/w/cpp/filesystem/canonical) first, then check if the path starts with root/userfiles ?

Reply:
Yes, but this will need including std::filesystem. I'm not sure if this header should create that dependency.

Reply:
can you move it to a separate header file then?

Reply:
updated

Reply:
OK, I'm not familiar with filesystem APIs in C++, so I'm going to trust you that this is working as intended.